Source/Initialization/ExternalField_fwd.H

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2023 Luca Fedeli
+ *
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef EXTERNAL_FIELD_FWD_H_
+#define EXTERNAL_FIELD_FWD_H_
+
+#include <AMReX_Parser.H>
```

Comment:
Are these includes needed?

Reply:
You are totally right! That's definitively a mistake! I will remove them immediately.